src/components/user-list.html

```diff
@@ -1,24 +1,54 @@
-<link rel="import" href="./user-profile.html">
+<link rel="import" href="../../node_modules/@banno/polymer/polymer.js">
+<link rel="import" href="../../node_modules/@banno/polymer/lib/elements/dom-repeat.js">
 <link rel="import" href="./user-profile.html">
 
 <dom-module id="user-list">
-    
-    <template>
-        <user-profile></user-profile>
-        <user-profile></user-profile>
-        <user-profile></user-profile>
-    </template>
+  <style>
+  </style>
+  <template>
+    <h2> User list: </h2>
+    <dom-repeat items="{{users}}">
+      <template>
+        <div>
+          <user-profile 
+            id="{{item.id}}" 
```

Comment:
One-way bind these, too

Reply:
Ah, just make all the changes I suggested in `user-list-test` here, and then get rid of the test component once this is good to go.